doc/data/messages/r/return-or-yield-in-finally/bad/yield.py

```diff
@@ -0,0 +1,11 @@
+def all_favorites():
+    fruits = ["orange", "kiwi", "pineapple"]
+
+    for fruit in fruits:
+        try:
+            yield fruit
+        finally:
+            # this `yield` statement will add `None` after every fruit
+            # list(all_favorites()) will result in:
+            # ["orange", None, "kiwi", None, "pineapple", None]
+            yield  # [return-or-yield-in-finally]
```

Comment:
Strikes me as a little contrived, who's to say that interleaving None between those fruits is not intended/useful? It's not a crazy use case to need to interleave a constant into a every other position in a sequence.